static/js/global.js

```diff
@@ -1,7 +1,7 @@
 var ai = [
   { url: "http://www.ubuntu.com", title:"Ubuntu" },
```

Comment:
what about this link? will we be https everywhere? I assume we have to change it AFTER we move to https?  Perhaps we should file and issue for that.

also, we need up update this file in the [new repo](https://github.com/ubuntudesign/global-nav) as well.